src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/Content/Sections/Customer/_billingAddress.html.twig

```diff
@@ -3,8 +3,8 @@
 {% set order = hookable_data.resource %}
 
 <div>
-    <strong>{{ 'sylius.ui.shipping_address'|trans }}:</strong>
+    <strong>{{ 'sylius.ui.billing_address'|trans }}</strong>
 </div>
-<div class="mb-3">
+<div class="mb-3" id="billing-address">
```

Comment:
```suggestion
<div class="mb-3" {{ sylius_test_html_attribute('billing-address') }}>
```
If the added `id` is intended for testing purposes, it is better to use `sylius_test_html_attribute` .